socorro/external/fs/filesystem.py

```diff
@@ -136,18 +139,18 @@ def findFileGenerator(
     """
     if 0 >= maxDepth:
         return
+    keep_dir = directoryAcceptanceFunction
```

Comment:
I added this to reduce the line length later. It's less work to do it inside the function rather than change the argument and adjust uses of this function elsewhere in the codebase.